android/app/src/main/kotlin/coop/polypoly/polypod/MainActivity.kt

```diff
@@ -17,37 +13,31 @@ import coop.polypoly.polypod.updatenotification.UpdateNotification
 class MainActivity : AppCompatActivity() {
 
     private var initialized = false
-    val desiredLockScreenType =
-        BiometricManager.Authenticators.BIOMETRIC_WEAK or
-            BiometricManager.Authenticators.DEVICE_CREDENTIAL
+
+    override fun onActivityResult(
```

Comment:
This doesn't do anything, right?

Reply:
Right. The plan was to send a message to MainActivity from OnboardingActivity, but this was never firing, so can be removed.